src/main/java/uk/gov/companieshouse/web/accounts/service/smallfull/impl/ApprovalServiceImpl.java

```diff
@@ -35,6 +37,22 @@
     private static final UriTemplate APPROVAL_URI =
             new UriTemplate("/transactions/{transactionId}/company-accounts/{companyAccountsId}/small-full/approval");
 
+    private static final String DAY_MONTH_REGEX = "[0-9]{1,2}";
+
+    private static final String YEAR_REGEX = "[0-9]{4}";
```

Comment:
Predefined character classes would work equally well here for both of these regular expression constants. For example `\d{4}`. See https://docs.oracle.com/javase/tutorial/essential/regex/pre_char_classes.html for reference. However, I wouldn't consider this a required change so feel free to leave as-is.